runtime/compiler/env/j9method.cpp

```diff
@@ -2387,15 +2393,48 @@ TR_ResolvedJ9Method::TR_ResolvedJ9Method(TR_FrontEnd * fe, TR_ResolvedMethod * o
 
 void TR_ResolvedJ9Method::construct()
    {
-#define x(a, b, c) a, sizeof(b) - 1, b, (int16_t)strlen(c), c
-
    struct X
       {
       TR::RecognizedMethod _enum;
       char _nameLen;
       const char * _name;
       int16_t _sigLen;
       const char * _sig;
+
+      typedef int8_t javaVersion_t;
+
+      javaVersion_t _nativeMinVersion;
+      javaVersion_t _nativeMaxVersion;
+
+// AllPastJavaVer and AllFutureJavaVer are used as lower and upper bounds for
+// ranges of Java versions over which a recognized method is expected to be
+// native, where that range is (currently) unbounded.  An upper bound of
+// AllPastJaveVer is used if the method has never been expected to be native.
+#define AllPastJavaVer   INT8_MIN
+#define AllFutureJavaVer INT8_MAX
+
+// xClMeth, xSig, xWCSig and xNonNativeRange are intermediate macros used in
+// the macros used to initialize entries in the tables of recognized methods
```

Comment:
Given these are not widely used could we just spell the names out to make reading this easier?